js/bitmart.js

```diff
@@ -798,7 +799,7 @@ module.exports = class bitmart extends Exchange {
         //          "legal_coin_price":"0.1302699"
         //      }
         //
-        const timestamp = this.safeTimestamp2 (ticker, 'timestamp', 's_t', this.milliseconds ());
+        const timestamp = this.safeTimestamp (ticker, 'timestamp', this.milliseconds ());
```

Comment:
@AlgoQ now, there's something going on with the timestamp 
![image](https://user-images.githubusercontent.com/43336371/199941478-0a055d90-48bd-4f17-b8e3-660c1fc52171.png)